pkg/kuttlctl/cmd/test.go

```diff
@@ -195,6 +193,7 @@ For more detailed documentation, visit: https://kuttl.dev`,
 			}
 
 			if len(args) != 0 {
+				log.Println("kutt-test config testdirs is overridden with arg: ", args)
```

Comment:
won't this end with a not nice looking log? Should we maybe rather do `strings.Join(args, ",")` ?

Reply:
+1. thanks @alenkacz !